packages/suite/src/components/suite/WebUsbButton.tsx

```diff
@@ -1,16 +1,23 @@
 import TrezorConnect from '@trezor/connect';
-import { Button, ButtonProps } from '@trezor/components';
-import { Translation } from 'src/components/suite';
+import { ButtonProps, IconButton } from '@trezor/components';
+import { useIntl } from 'react-intl';
+import messages from 'src/support/messages';
 
-export const WebUsbButton = (props: Omit<ButtonProps, 'children'>) => (
-    <Button
-        {...props}
-        icon="SEARCH"
-        onClick={e => {
-            e.stopPropagation();
-            TrezorConnect.requestWebUSBDevice();
-        }}
-    >
-        <Translation id="TR_CHECK_FOR_DEVICES" />
-    </Button>
-);
+export const WebUsbButton = (props: Omit<ButtonProps, 'children'>) => {
+    const intl = useIntl();
```

Comment:
Only `useTranslation` should be used everywhere for better type safety. With useIntl one can write `intl.formatMessage({id: 'foo', defaultMessage: 'baf'})` with no type warning. I also find it more convenient to only import `useTranslation` instead of importing both useIntl and messages....

Reply:
Yes, this looks better, I didn't find it before. Maybe we should ideally delete all occurences of `const intl = useIntl();` because there are some of them.

Reply:
It's used in `components` [as well](https://github.com/trezor/trezor-suite/blob/0f502e575065bd1eb1dc233670389e509a818b29/packages/components/src/components/Passphrase/PassphraseTypeCard.tsx#L173), since messages are in the in suite, so there IS a use for it 😅